src/OpenColorIO/NamedTransform.cpp

```diff
@@ -240,7 +240,7 @@ std::ostream & operator<< (std::ostream & os, const NamedTransform & t)
 {
     os << "<NamedTransform ";
     const std::string strName{ t.getName() };
-    os << "name=" << strName;
+    os << "name=" << strName << ", ";
```

Comment:
If there is no aliases the line 261 prepend a comma which conflicts with the comma added here. The comma logic in the method is to always prepend a comma (and not append it). So, the comma error is in the `alias` code block.